ftw/publisher/controlling/browser/views.py

```diff
@@ -5,11 +5,12 @@
 from Products.statusmessages.interfaces import IStatusMessage
 from ftw.publisher.controlling import _
 from ftw.publisher.controlling.interfaces import IStatisticsCacheController
+from ftw.publisher.sender.workflows import interfaces
```

Comment:
Is this file imported at frontend side?
Be aware that we do not have `ftw.publisher.sender` on the frontend installed; so we might need to make that safe.
Be also aware that you probably have a shared installation locally, which have both, sender and receiver installed.

Reply:
@jone, I have pushed a new commit which separates the code related to sender and receiver. Could you please have another look?